sdk/monitor/azquery/metrics_client_test.go

```diff
@@ -10,11 +10,34 @@ import (
 	"context"
 	"testing"
 
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
 	"github.com/Azure/azure-sdk-for-go/sdk/monitor/azquery"
 	"github.com/stretchr/testify/require"
 )
 
+func TestMetricsClient(t *testing.T) {
+	client, err := azquery.NewMetricsClient(credential, nil)
+	require.NoError(t, err)
+	require.NotNil(t, client)
+
+	c := cloud.Configuration{
+		ActiveDirectoryAuthorityHost: "https://...",
+		Services: map[cloud.ServiceName]cloud.ServiceConfiguration{
+			cloud.ResourceManager: {
+				Audience: "",
+				Endpoint: "",
+			},
+		},
+	}
+	opts := azcore.ClientOptions{Cloud: c}
+	cloudClient, err := azquery.NewMetricsClient(credential, &azquery.MetricsClientOptions{ClientOptions: opts})
+	require.Error(t, err)
+	require.Equal(t, err.Error(), "provided Cloud field is missing Azure Monitor Metrics configuration")
```

Comment:
FYI, not that it's a big improvement, but there's a helper for this:
```suggestion
	require.EqualError(t, err, "provided Cloud field is missing Azure Monitor Metrics configuration")
```